public/app/plugins/panel/gettingstarted/GettingStarted.tsx

```diff
@@ -137,34 +79,114 @@ export class GettingStarted extends PureComponent<PanelProps, State> {
   };
 
   render() {
-    const { checksDone } = this.state;
+    const { checksDone, currentStep, steps } = this.state;
+
     if (!checksDone) {
       return <div>checking...</div>;
```

Comment:
Getting this on a slow connection:

![image](https://user-images.githubusercontent.com/2376619/81566551-51cfac80-939b-11ea-9f7f-670791dee918.png)